modules/addTodoList.js

```diff
@@ -0,0 +1,28 @@
+import { todocontainer } from './variables.js';
+import deleteItem from './removeTodo.js';
+import edit from './edit.js';
+
```

Comment:
- [ ] By default, new tasks should have the property completed set to false and the property index set to the value of the new array length